src/game/server/neo/bot/neo_bot_locomotion.cpp

```diff
@@ -20,7 +20,13 @@ void CNEOBotLocomotion::Update( void )
 	// always 'crouch jump'
 	if ( IsOnGround() )
 	{
+#ifdef NEO
+		// NEO JANK resetting of crouch timer moved to NextBotPlayer::PressJumpButton
+		// so far crouch jump seems to still be working, but watch out for a regression
+		// disabled:
+#else
 		me->ReleaseCrouchButton();
+#endif
```

Comment:
Did you mean to ifdef-guard the `ReleaseCrouchButton` to never execute here? Regardless, we are inside a neo-specific file here, so we should not be using `ifdef NEO`. That's only to be used when writing our code inline with Valve's base engine code.